src/main/java/com/microsoft/sqlserver/jdbc/InformationType.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.sqlserver.jdbc;
```

Comment:
`InformationType` and `Label` classes are exactly the same. Please also make sure the class names are descriptive.

Reply:
We cannot rename classes, as they are as per design specs.

On the other hand, I think it would be better to move all these new classes to `dataclassification` package so that customers would relate `Label` and` InformationType` classes to data classification information and not mix with other driver classes.

Other drivers have implemented differently with all classes defined in a common class `DataClassification `, but in Java we shall do in separate individual classes as they are public APIs.